python_modules/dagster/dagster_tests/core_tests/storage_tests/test_event_log.py

```diff
@@ -33,8 +32,14 @@ class TestSqliteEventLogStorage(TestEventLogStorage):
 
     @pytest.fixture(scope="function", name="storage")
     def event_log_storage(self):  # pylint: disable=arguments-differ
-        with tempfile.TemporaryDirectory() as tmpdir_path:
-            yield SqliteEventLogStorage(tmpdir_path)
+        # make the temp dir in the cwd since default temp roots
+        # have issues with FS notif based event log watching
+        with tempfile.TemporaryDirectory(dir=os.getcwd()) as tmpdir_path:
+            try:
+                storage = SqliteEventLogStorage(tmpdir_path)
+                yield storage
+            finally:
+                storage.dispose()
```

Comment:
could stick an "if storage" in here or more the initialization outside the try - if the constructor fails, storage will be undefined